wagtail/admin/views/account.py

```diff
@@ -312,6 +312,16 @@ def get(self, *args, **kwargs):
     def get_form_class(self):
         return get_user_login_form()
 
+    def form_valid(self, form):
+        login(self.request, form.get_user())
+        remember = form.cleaned_data.get('remember')
+        if remember:
+            self.request.session.set_expiry(settings.SESSION_COOKIE_AGE)
+        else:
+            self.request.session.set_expiry(0)
+
+        return HttpResponseRedirect(self.get_success_url())
+
```

Comment:
Suggestion: I wonder if you could save some imports and duplication?

```suggestion
    def form_valid(self, form):
        response = super().form_valid(form)

        remember = form.cleaned_data.get('remember')
        if remember:
            self.request.session.set_expiry(settings.SESSION_COOKIE_AGE)
        else:
            self.request.session.set_expiry(0)

        return response

```

You may be able to put the `super` call at the end too, I'm not sure

Reply:
Yes yours is nicer. Can't call `super` at the end because it resets the session.